embassy-stm32/src/rcc/g4.rs

```diff
@@ -512,3 +512,12 @@ pub(crate) unsafe fn init(config: Config) {
         apb2_tim: Hertz(apb2_tim_freq),
     });
 }
+
+pub(crate) unsafe fn enable_hsi48() {
```

Comment:
Need to look at which other chips have the CRS module and add this to them as well.

Reply:
Since the CRS work is moved into the g4 RCC module I'm not going to tackle this right now. If someone has a G0 device with USB that I could test with I would be willing to port this over.